src/lib/db.js

```diff
@@ -1578,3 +1442,110 @@ export function getSemesterTeams(semesterName, teamIndices) {
     })
   ));
 }
+
+export async function getNumArticles() {
+  const rows = await database('articles').count('* as numArticles');
+  return rows[0].numArticles;
+}
+
+/**
+ * Updates which category an article belongs to
+ * @param {Object} jsonGraphArg - An object of type { [slugs]: articleUpdateObject[] }
```

Comment:
Also I think this documentation is just wrong, it's not an array of objects, just an object. And I shouldn't pluralize `slugs`